### PR TITLE
Minor fix by Revotix (NL)

### PR DESCRIPTION
**Pull request by revotix**
REVOTIX#0001

### CODE REVIEW

**Comment by revotix** [comment]
I changed some minor things and fixed some things.

File: nl.json [hunk 1]
```diff
@@ -1,192 +1,192 @@
...
+            "view" : {
+                "title" : "Winkel Rotatie",
+                "line1" : "Dit is de item winkel rotatie van [date] voor Fortnite Battle Royale. ",
+                "line2" : "Klik op een item om meer informatie te zien.",
+                "today-button" : "Item Winkel van vandaag"
+            }
+        },
+        "news" : {
+            "title" : "Nieuws",
+            "line1" : "De huidige in-game nieuws feed van Epic Games. Dit wordt elke minuut geüpdate.",
```
Comment (Byte-Wolf) [inline nl.json]: De correcte vorm is toch echt geüpdatet, zoek het maar op!

Comment (revotix) [reply nl.json]: > De correcte vorm is toch echt geüpdatet, zoek het maar op!

Oeps sorry, kan gebeuren!

Comment (Byte-Wolf) [reply nl.json]: Maakt niets uit, dan weet je dat voor de volgende keer! We maken allemaal wel eens fouten.

**Comment by Krymonota** [comment]
In case you're wondering: There's a conflict that must be resolved first so that this pull request can be merged. 

In simple terms, the problem is that the `nl.json` file has been updated in the meantime and your changes would conflict with the update(s). If you want your pull request to be accepted, you have to fix your pull request if no one else is taking care of the problem for you.

**Comment by njb-said** [comment]
Done in https://github.com/fnbrco/translations/commit/e87dc22ec58357253895d3396aebcdf089b54097

**Pull request closed without merging**